models/ios/logging_global/ios_logging_global.yaml

```diff
@@ -0,0 +1,1071 @@
+                              ipv4: 
+                                description: Include logging source IP address in session ID tag
+                                type: bool
+                              ipv6: 
+                                description: Include logging source IPV6 address in session ID tag
+                                type: bool
+                              string: 
+                                description: Include custom string in session ID tag
+                                type: str
+                          xml: 
+                            description:  Enable logging in XML
+                            type: dict
+                            suboptions:
+                              discriminator: 
+                                description:  Specify a message discriminator identifier for this logging session
+                                type: str
+                              sequence_num_session: 
+                                description:  Include session sequence number tag in syslog message
+                                type: dict
+                                suboptions:
+                                  discriminator: 
+                                    description:  Specify a message discriminator identifier for this logging session
+                                    type: str
+                              session_id: 
+                                description:  Specify syslog message session ID tagging
+                                type: dict
+                                suboptions:
+                                  hostname: 
+                                    description: Include hostname in session ID tag
+                                    type: dict
+                                    suboptions:
+                                      discriminator: 
+                                        description:  Message discriminator identifier for this logging session
+                                        type: str
+                                      sequence_num_session: 
+                                        description:  Message discriminator identifier for this logging session
+                                        type: dict
+                                        suboptions:
+                                          discriminator: 
+                                            description:  Specify a message discriminator identifier for this logging session
+                                            type: str
+                                  ipv4: 
+                                    description: Include logging source IP address in session ID tag
+                                    type: bool
+                                  ipv6: 
+                                    description: Include logging source IPV6 address in session ID tag
+                                    type: bool
+                                  string: 
```

Comment:
same as above `string` comment

Reply:
@justjais  Yes, thought of it, any suggestions I would make it `text:` anything specific that we follow, moreover it should remember the command to the user right!